app/jobs/notifications/milestone_job.rb

```diff
@@ -0,0 +1,12 @@
+module Notifications
+  class MilestoneJob < ApplicationJob
+    queue_as :send_milestone_notification
+
+    def perform(milestone_hash, service = Notifications::Milestone::Send)
+      article = Article.find_by(id: milestone_hash[article])
+      return unless article
+
+      service.call(milestone_hash["type"], article)
```

Comment:
The hash is passed with the symbol keys but the value is retrieved by a string key.

Reply:
Related to the same typo above. I have corrected.